minijinja/src/value/mod.rs

```diff
@@ -853,10 +904,15 @@ impl Value {
                 m.iter()
                     .filter_map(|(k, v)| k.as_str().map(move |k| (k, v.clone()))),
             ) as Box<dyn Iterator<Item = _>>,
-            ValueRepr::Dynamic(ref obj) => Box::new(
-                obj.attributes()
-                    .filter_map(move |attr| Some((attr, some!(obj.get_attr(attr))))),
-            ) as Box<dyn Iterator<Item = _>>,
+            ValueRepr::Dynamic(ref obj) => match obj.behavior() {
+                ObjectBehavior::Basic | ObjectBehavior::Seq(_) => {
+                    Box::new(None.into_iter()) as Box<dyn Iterator<Item = _>>
```

Comment:
Note that there's `std::iter::empty()`.

Reply:
I might change this in the future. For now this is a pattern that is used in a few places.